scenario.py

```diff
@@ -75,56 +78,50 @@ def __init__(self, params, experiment_path):
         self.contributivity_list = []
 
         # Number of epochs and mini-batches in ML training
-        if 'epoch_count' in params:
-            self.epoch_count = params['epoch_count']
-            assert self.epoch_count > 0 
+        if "epoch_count" in params:
+            self.epoch_count = params["epoch_count"]
+            assert self.epoch_count > 0
         else:
             self.epoch_count = 40
-        
-        if 'minibatch_count' in params:
-            self.minibatch_count = params['minibatch_count']
+
+        if "minibatch_count" in params:
+            self.minibatch_count = params["minibatch_count"]
             assert self.minibatch_count > 0
         else:
             self.minibatch_count = 20
-            
+
         # Contributivity methods
         ALL_METHODS_LIST = [
             "Shapley values",
-            "Independant scores", 
+            "Independant scores",
             "TMCS",
             "ITMCS",
             "IS_lin_S",
             "IS_reg_S",
             "AIS_Kriging_S",
             "SMCS",
-            "WR_SMC"
-            ]
-        
-        # List of Contributivity methods runned by default if no method was given in the config file 
-        DEFAULT_METHODS_LIST = [
-            "Shapley values",
-            "Independant scores",
-            "TMCS"
-            ]
-        
+            "WR_SMC",
+        ]
+
+        # List of Contributivity methods runned by default if no method was given in the config file
+        DEFAULT_METHODS_LIST = ["Shapley values", "Independant scores", "TMCS"]
+
         self.methods = []
-        if 'methods' in params:
-            if params['methods']:
-                for el in params['methods']:
+        if "methods" in params:
+            if params["methods"]:
+                for el in params["methods"]:
```

Comment:
I think this could be reduced, maybe like `if "methods" in params and params["methods"]` (but isn't it the same check?), what do you think about it?

Reply:
Yes I will change that.

- `"methods" in params`: check if methods is in dict
- ` params["methods"]`: check if value is not null

Reply:
indeed! #need_coffee